data-pipeline/src/pipeline/input_schemas.py

```diff
@@ -262,146 +262,122 @@
-    "Premises staff": "float",
-    "Catering staff": "float",
-    "Other staff": "float",
-    "Indirect employee expenses": "float",
-    "Staff development and training": "float",
-    "Staff-related insurance": "float",
-    "Supply teacher insurance": "float",
-    "Building and Grounds maintenance and improvement": "float",
-    "Cleaning and caretaking": "float",
-    "Water and sewerage": "float",
-    "Energy": "float",
-    "Rent and Rates": "float",
-    "Other occupation costs": "float",
-    "Special facilities": "float",
-    "Learning resources (not ICT equipment)": "float",
-    "ICT learning resources": "float",
-    "Examination fees": "float",
-    "Educational Consultancy": "float",
-    "Administrative supplies - non educational": "float",
-    "Agency supply teaching staff": "float",
-    "Catering supplies": "float",
-    "Other insurance premiums": "float",
-    "Legal & Professionalservices": "float",
-    "Auditor costs": "float",
-    "Interest charges for Loan and Bank": "float",
-    "Direct revenue financing - Revenue contributions to capital": "float",
-    "PFI Charges": "float",
-    "Revenue reserve": "float",
-    "Total Grant Funding": "float",
-    "Direct Grants": "float",
-    "Community Grants": "float",
-    "Targeted Grants": "float",
-    "Total Self Generated Funding": "float",
-    "Total Income": "float",
-    "Supply Staff Costs": "float",
-    "Other Staff Costs": "float",
-    "Total Staff Costs": "float",
-    "Maintenance & Improvement Costs": "float",
-    "Premises Costs": "float",
-    "Catering Expenses": "float",
-    "Occupation Costs": "float",
-    "Total Costs of Supplies and Services": "float",
-    "Total Costs of Educational Supplies": "float",
-    "Costs of Brought in Professional Services": "float",
-    "Total Expenditure": "float",
-    "London Weighting": "string",
-    "Date joined or opened if in period": "string",
-    "Date left or closed if in period": "string",
+    "ACADEMYUPIN": "Int64",
+    "Company_Number": "string",
+    "Date joined or opened if in period:": "string",
+    "Date left or closed if in period:": "string",
+    "BNCH11110T (EFA Revenue Grants)": "float",
+    "BNCH11131 (DfE Family Revenue Grants)": "float",
+    "BNCH11141 (SEN)": "float",
+    "BNCH11142 (Other Revenue)": "float",
+    "BNCH11151 (Other Government Revenue Grants)": "float",
+    "BNCH11161 (Government source (non-grant))": "float",
+    "BNCH11162 (Academies)": "float",
+    "BNCH11163 (Non- Government)": "float",
+    "BNCH11123-BAI011-A (Academies - Income)": "float",
+    "BNCH11201 (Income from facilities and services)": "float",
+    "BNCH11202 (Income from catering)": "float",
+    "BNCH11203 (Receipts from supply teacher insurance claims)": "float",
+    "BNCH11300T (Voluntary income)": "float",
+    "BNCH11204 (Other income - revenue)": "float",
+    "BNCH11205 (Other Income from facilities and services)": "float",
+    "BNCH11400T (Investment income)": "float",
```

Comment:
Do we know what the difference is between `BNCH11401 (Investment income)` and `BNCH11400T (Investment income)`?